EventListener/SecurityListener.php

```diff
@@ -79,7 +85,8 @@ private function getVariables(Request $request)
             'request' => $request,
             'roles' => array_map(function ($role) { return $role->getRole(); }, $roles),
             'trust_resolver' => $this->trustResolver,
-            'security_context' => $this->securityContext,
+            // needed for the is_granted expression function
+            'auth_checker' => $this->authChecker,
```

Comment:
ah, I though the parent class in the Security component was also using this variable. But it indeed does not